_static/css/custom_1582183857228.css

```diff
@@ -235,3 +233,20 @@ redoc .param-name {
 	word-break: break-all;
 	white-space: normal !important;
 }
+
+body {
+    overflow-x: initial !important;
```

Comment:
@JacekClearcode It's standard upstream merge. Is there a reason why v10.1 doesn't use new redoc, or can I merge it safely?

Reply:
As we spoke privately, you can merge this safely.